src/main/java/com/lginterview/controllers/CourseController.java

```diff
@@ -0,0 +1,36 @@
+package com.lginterview.controllers;
+
+import com.lginterview.dto.CourseRequest;
+import com.lginterview.dto.CourseStatus;
+import com.lginterview.exceptions.CourseDataRetrievalException;
+import com.lginterview.providers.CourseProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class CourseController {
+
+    private final CourseProvider courseProvider;
+
+    @Autowired
+    public CourseController(CourseProvider courseProvider) {
+        this.courseProvider = courseProvider;
+    }
+
+    @RequestMapping(path = "/status/course", method = RequestMethod.POST)
+    public ResponseEntity<Long> modifyCourse(@RequestBody @Validated CourseRequest courseRequest)
+            throws CourseDataRetrievalException {
+        return new ResponseEntity<>(courseProvider.modifyCourse(courseRequest), HttpStatus.OK);
```

Comment:
[MS][Minor] You're always returning HTTP 200, nevertheless the CourseRequest created the new Course (which should be 201), returned the same (probably 200) or updated/finished (probably 202 Accepted).

Reply:
I Agree